### PR TITLE
XW-447 | Additional changes to Curated Content Editor

### PR DESCRIPTION
**Pull request by vforge**
https://wikia-inc.atlassian.net/browse/XW-447

Best viewed https://github.com/Wikia/mercury/pull/1456/files?w=0

### CODE REVIEW

**Comment by bognix** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

File: front/scripts/main/mixins/CuratedContentEditorLabelsMixin.ts [hunk 1]
```diff
@@ -18,5 +23,13 @@ App.CuratedContentEditorLabelsMixin = Em.Mixin.create({
 				return i18n.t('app.curated-content-editor-new-category');
 			}
 		}
+	}),
+
+	pageNameLabel: Em.computed('isCategory', function(): string {
```
Comment (bkoval) [inline front/scripts/main/mixins/CuratedContentEditorLabelsMixin.ts]: Well, it seems that whitespacing here is different than e.g. here -> https://github.com/Wikia/mercury/pull/1456/files#diff-b01dd9254a9968a270312b97a860ed93R5 I gotta say I'm dissapointed @rogatty didn't notice that :)

Comment (rogatty) [inline front/scripts/main/mixins/CuratedContentEditorLabelsMixin.ts]: :speak_no_evil:

**Comment by rogatty** [comment]
LGTM

**Pull request merged**